Zend/tests/pipe_operator/precedence_addition.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+Pipe binds lower than addition
```

Comment:
The test as written does not reliably verify that, because both:

```
5 + (2 |> '_test1')
```
and
```
(5 + 2) |> '_test1'
```
result in 8.

Reply:
(also, what's up with the underscores in the function names?)

Reply:
> (also, what's up with the underscores in the function names?)

I probably had a reason 4 years ago when I first wrote these tests, but I couldn't tell you why, now.  I was probably copying something else at the time.